plugins/default/src/Ide/Plugin/Pragmas.hs

```diff
@@ -142,13 +142,13 @@ completion lspFuncs _ide complParams = do
         position = complParams ^. J.position
     contents <- LSP.getVirtualFileFunc lspFuncs $ toNormalizedUri uri
     fmap Right $ case (contents, uriToFilePath' uri) of
-        (Just cnts, Just _path) -> do
-            pfix <- VFS.getCompletionPrefix position cnts
-            return $ result pfix
+        (Just cnts, Just _path) ->
+            result <$> VFS.getCompletionPrefix position cnts
             where
                 result (Just pfix)
                     | "{-# LANGUAGE" `T.isPrefixOf` VFS.fullLine pfix
```

Comment:
I note some people use `{-# language` - or any other case alternative. I'm happy for those people not to get completions though...

Reply:
True, and I think this is a problem with other pragmas not just LANGUAGE. However, this is orthogonal to this PR, I did not write this code. If someone cares deeply, they'll open an issue and/or send a PR.

Reply:
I'd rather make GHC enforce a case-sensitive pragma (and if you have to, upper case is surely it). I find it super weird Haskell is mostly case sensitive apart from when its not.